flexget/plugins/sites/divxatope.py

```diff
@@ -31,14 +31,13 @@ class UrlRewriteDivxATope(object):
     # urlrewriter API
     def url_rewritable(self, task, entry):
         url = entry['url']
-        return (
-            url.startswith('http://www.divxatope.com/descargar') or url.startswith('http://divxatope.com/descargar')
-        )
+        rewritable_regex = '^http:\/\/(www.)?divxatope1?.com.*'
```

Comment:
Their certificate is expired, but I will add the "https?" with the other changes.